app/templates/components/user-sidebar.hbs

```diff
@@ -1,13 +1,21 @@
 <img src={{user.photoLargeUrl}} />
 <h2>
-  <div class="name">{{user.name}}</div>
+  {{#if user.name}}
+    <div class="name">{{user.name}}</div>
+  {{else}}
+    <div class="name">{{user.username}}</div>
+  {{/if}}
   <div class="username">{{user.username}}</div>
```

Comment:
This will cause a duplicate display of the username in the case where the user has no actual name:

```HTML
<div class="name">username</div>
<div class="username">username</div>
```